docs/source/developer_resources.rst

```diff
@@ -426,21 +410,50 @@ Older versions of the source code documentation are (still) on readthedocs; howe
 
 For releasing, copy the ``htmlout`` directory into the seed-platform's website repository under ``docs/code_documentation/<new_version>``. Make sure to add the new documentation to the table in the ``docs/developer_resources.md``.
 
+Contribution Instructions / Best Practices
+------------------------------------------
+
+If this is the first time contributing and you are outside of the DOE National Lab system, then you will need to review and fill out the contribution agreement which is found in `SEED's Contribution Agreement in the GitHub repository`_
```

Comment:
Yeah, there are directions on the link on how to "agree to the seed contribution policy". The link will be there when rendered on the website, but since it is in markdown, the link is on the bottom of that page (https://github.com/SEED-platform/seed/blob/develop/.github/CONTRIBUTING.md).